App.svelte

```diff
@@ -74,6 +74,10 @@
     required: {
       keywords: ['required'],
```

Comment:
Add this as another keyword here
```suggestion
      keywords: ['required', 'requirement'],
```